### ISSUE TITLE
Essentials MOTD is ran before login 

### ISSUE THREAD

**Issue by Eufranio** [AuthMe/AuthMeReloaded#1042]
I would like that you guys delay a bit the Essentials MOTD command, like 1 second or so, it is ran before the player actually login, then the logged in message shows after the MOTD...

**Comment by ljacqu** [comment]
This is basically the opposite of the request in #935 :P

**Comment by Eufranio** [comment]
You guys could create a expression in the commands.yml file. That way, it would be:
```
onLogin:
    welcome:
        delay 1 second|wait 1 second|wait 1s|wait 1sec
        command: 'motd'
        executor: 'PLAYER'
```

Then you could remove the config that runs the Essentials motd on join from the main config.

**Comment by ljacqu** [comment]
Oh right, motd is on login. Totally agree with you that we should move it to the commands.yml. An optional delay property would be great.

**Comment by Twonox** [comment]
Same thing for the Session's message. (it happen sometimes, not always)
<a href="http://www.noelshack.com/2017-01-1483653314-2017-01-05-225253.png"><img src="http://image.noelshack.com/minis/2017/01/1483653314-2017-01-05-225253.png" border="0" alt="1483653314-2017-01-05-225253.png - envoi d'image avec NoelShack" title="1483653314-2017-01-05-225253.png"/></a>

**Comment by techxiu** [comment]
But Motd looks like can't set not shown in the joining.If disabling motd command,it also can't shown when player enter motd,

**Comment by ljacqu** [comment]
If we solve this by implementing an optional delay, we can close #1449 as well.

**Comment by ljacqu** [comment]
We missed that this was solved with the introductions of delays (#1449)
